addon/templates/components/lt-body.hbs

```diff
@@ -18,6 +20,7 @@
         {{else}}
           {{#each rows as |row|}}
             {{lt.row row columns
+              elementId=row.elementId
```

Comment:
Actually *has* to be named [`elementId`](http://emberjs.com/api/classes/Ember.Component.html#property_elementId).